doc/doxygen/src/ForHosts.dox

```diff
@@ -98,21 +98,32 @@
  *   user with an additional manual configuration mechanism for your
  *   specific API session if you so wish.
  *
- * - Create, persist and configure a @ref Context as appropriate for
- *   all calls to the API.
+ * - Create (and configure) a @ref Context as appropriate for
+ *   all calls to the API. The Context instance should be kept alive
+ *   and re-used for API calls that are related to the same user
+ *   session, or logical set of interactions. For example, one per
+ *   document. Use @fqref{hostApi.Manager.createChildContext}
+ *   "createChildContext" to split off a new Context should you need to
+ *   re-configure it for any specific series of actions, whilst
+ *   maintaining the association.
  *
  * - Ask the @ref manager if they support resolve for the relevant
  *   data by supplying its @ref trait_set to
  *   @fqref{hostApi.Manager.managementPolicy} "Manager.managementPolicy"
  *   with a @ref Context with `kRead` access, respecting the returned
  *   traits - this may be a subset if the manager is only capable of
- *   resolving some of an entity's traits.
+ *   resolving some of an entity's traits. Note that policy is
+ *   runtime invariant for any given context/access/trait set.
```

Comment:
[Question] What's the action that a implementor should take in response to this note? I suppose you saw this come up somewhere, but I'm struggling to see what the outcome is, is it like, you don't need to test the management policy every time for the same trait set?

Edit : Ah yes, i see you've outlined that below, I wonder if the docs from there would also fit here. This is a checklist and the things in it should be actionable.

Reply:
[consider][minor] Trying to keep phrasing consistent.
```suggestion
 *   resolving some of an entity's traits. Note that managementPolicy is
 *   runtime invariant for any given set of inputs.
```

Reply:
The action is "Ask the @ref manager if they support resolve...", the note was just providing more context whilst we had their attention, in case they didn't read the rest of the docs.

Re details vs 'inputs', as we dont describe the signature in this para, it felt like it helped give a more comprehensive picture of the invariance, that didn't need a follow up investigation. So the reader may hopefully come away with a deeper understanding of the underlying concept if this is all `they.`